shotover-proxy/tests/test-topologies/tee/subchain.yaml

```diff
@@ -0,0 +1,23 @@
+---
+sources:
+  redis_prod:
+    Redis:
+      listen_addr: "127.0.0.1:6379"
+      connection_limit: 3000000
+chain_config:
+  redis_chain:
+    - Tee:
+        behavior:
+          SubchainOnMismatch:
+            - DebugReturner:
+                Redis: "42"
```

Comment:
Dont we need to hit a real redis instance so we can assert that the subchain isnt run?